engine/exception.go

```diff
@@ -329,7 +353,12 @@ func (ev exceptionalValue) Term() Term {
 	return exceptionalValueAtoms[ev]
 }
 
+// EvaluationError creates a new evaluation error exception.
+func EvaluationError(ev Term, env *Env) Exception {
```

Comment:
Shouldn't `ev` have a more descriptive name since it's exposed?

ISO/IEC 13211-1 7.12.2.g says an evaluation error takes form of `evaluation_error(Error)` so why don't we make it `func EvaluationError(error Term, env *Env) Exception`?

Reply:
Yes, I completely agree. I'll make the change. :relieved: